### PR TITLE
Make the image and mask outputs work for a batch of images or masks

### PR DESCRIPTION
**Pull request by claussteinmassl**
Refactor tensor concatenation to use torch.stack instead of torch.cat in face_parsing_nodes.py to make it work for a batch of images or masks.

Behaviour before:
Given a batch of images as input, e.g. the FaceParser node would return a single image with all images inside it due to the way the output tensor was buuld.

Behaviour now:
A batch of images or masks is returned, making it compatible with other ComfyUI Nodes.

### CODE REVIEW

**Comment by Ryuukeisyou** [comment]
thank you! i have added the fixes now.
